<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCSX2Steam</class>
 <widget class="QMainWindow" name="PCSX2Steam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PCSX 2 Steam</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(51, 51, 51);
border-style: 0px; 
/*background-image: url(&quot;:/res/imgs/bg.png&quot;);*/

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="logoDisplay">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>318</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QGraphicsView#logoDisplay
{
	background-image: url(&quot;:/res/imgs/logo.png&quot;);
	/*border-style: 0px; */ 
}
</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameInput">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>205</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ccc;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="iconInput">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>205</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ccc;</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>103</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #45b9e2;</string>
    </property>
    <property name="text">
     <string>NAME</string>
    </property>
   </widget>
   <widget class="QLabel" name="iconLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>153</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #45b9e2;</string>
    </property>
    <property name="text">
     <string>ICON</string>
    </property>
   </widget>
   <widget class="QLabel" name="dirLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>203</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #45b9e2;</string>
    </property>
    <property name="text">
     <string>PATH</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dirInput">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>205</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ccc;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dirBrowseBtn">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>200</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#dirBrowseBtn
{
   background-color: #cccccc;
}
QPushButton:hover#dirBrowseBtn
{
  background-color: #f1f1f1;
}
QPushButton:pressed#dirBrowseBtn
{
   background-color: #45b9e2;
}</string>
    </property>
    <property name="text">
     <string>BROWSE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#createButton
{
   background-color: #cccccc;
}
QPushButton:hover#createButton
{
  background-color: #f1f1f1;
}
QPushButton:pressed#createButton
{
   background-color: #45b9e2;
}</string>
    </property>
    <property name="text">
     <string>CREATE</string>
    </property>
   </widget>
   <widget class="QLabel" name="dirLabel_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #171717;</string>
    </property>
    <property name="text">
     <string>D E S I G N E D   A N D   D E V E L O P E D   B Y   R A M B O R I G S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="iconPicBtn">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>100</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#iconPicBtn
{
    background-image: url(&quot;:/res/imgs/blankico.png&quot;);
    background-color: #cccccc;
}
QPushButton:hover#iconPicBtn
{
  background-color: #f1f1f1;
  background-image: url(&quot;:/res/imgs/blankico2.png&quot;);
}
QPushButton:pressed#iconPicBtn
{
   background-color: #45b9e2;
   background-image: url(&quot;:/res/imgs/blankico3.png&quot;);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#resetButton
{
   background-color: #cccccc;
}
QPushButton:hover#resetButton
{
  background-color: #f1f1f1;
}
QPushButton:pressed#resetButton
{
   background-color: #45b9e2;
}</string>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
